lib/resources/example_json/r4_examples_slot.rb

```diff
@@ -243,5 +333,77 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_SLOT_PATIENT_ACCESS_BUNDLE_INCLUDE_SCHEDULE ||= {
+      'resourceType': 'Bundle',
+      'id': 'f8dd806e-62bb-4657-8912-9c2705864d40',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Slot?service-type'\
+                '=https%3A%2F%2Ffhir.cerner.com%2Fec2458f2-1e24-41c8-b71b-0e701af7583d%2FcodeSet%2F14249%7C'\
+                '4047611&start=ge2016-01-23T22%3A05%3A47Z&start=lt2020-06-08T18%3A13%3A55Z&_include=Slot%3A'\
+                'schedule&_count=50'
+        },
+        {
+          'relation': 'next',
+          'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Slot?_include'\
+                '=Slot%3Aschedule&-pageContext=2d46d307-9a4c-4dc7-a1c2-88f1282d5aec&-pageDirection=NEXT'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Slot/4047611-32216049-61518614-0',
+          'resource': R4_SLOT_PATIENT_ACCESS_ENTRY
+        },
+        {
+          'fullUrl': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Schedule/4047611-25442717-61519334-45',
```

Comment:
nit: Shouldn't Schedule ID be the same as slot?

Reply:
addressed